### PR TITLE
add SCAN kiosk project

### PR DESCRIPTION
**Pull request by jccraft-uw**

### CODE REVIEW

**Comment by jccraft-uw** [comment]
[Update SCAN Switchboard to include new Greek system project](https://trello.com/c/o887tpij/675-update-scan-switchboard-to-include-new-greek-system-project)

File: bin/export-record-barcodes [hunk 1]
```diff
@@ -57,6 +57,7 @@ def main():
         ScanProject(22475, "es", "irb"),
         ScanProject(22476, "ko", "irb"),
         ScanProject(22477, "vi", "irb"),
+        ScanProject(23089, "en", "irb_kiosk"),
```
Comment (kairstenfay) [inline bin/export-record-barcodes:60]: the purview should use a hyphen as a delimiter, not an underscore. See https://github.com/seattleflu/id3c-customizations/pull/118

**Comment by kairstenfay** [comment]
Ah sorry, I was mistaken. The Switchboard is not dependent on ID3C-customizations, but there is still a conflict in the token naming between the two PRs that needs to be addressed. Stylistically, the id3c-customizaitons name (hyphen delimited) is more consistent, so that's my preference.

File: bin/export-record-barcodes (hunk 1 above)
Comment (jccraft-uw) [reply bin/export-record-barcodes]: Thanks for that catch! I fixed it to match Jover's PR.

Comment (kairstenfay) [reply bin/export-record-barcodes]: Thank you!

**Pull request merged**